django/db/migrations/operations/base.py

```diff
@@ -99,15 +99,15 @@ def references_field(self, model_name, name, app_label=None):
         """
         return self.references_model(model_name, app_label)
 
-    def allowed_to_migrate(self, connection_alias, model, hints=None):
+    def allow_migrate_model(self, connection_alias, model):
```

Comment:
This rename is not strictly needed but I'd like to make it more explicit that this is just a thin wrapper around `router.allow_migrate_model`. Let me know if you think this is overly invasive (it's a private API).

I thought of moving the `if model._meta.proxy or model._meta.swapped or not model._meta.managed` logic to `router.allow_migrate_model` and do away with this method entirely but `loaddata` (ab)uses `router.allow_migrate_model` to check if the model is allowed for a given database. I guess we should make it more explicit that we expect routers to support proxy/unmamanged/swapped models even though migrations won't migrate them.